project_addon/project_addon/doctype/timesheet_record/timesheet_record.py

```diff
@@ -63,23 +42,10 @@ def create_timesheet(self):
 				"task": self.task
 			},
 		)
-		timesheet.employee = self.employee
-
 		timesheet.insert()
-		self.db_set('timesheet', timesheet.name)
```

Comment:
In reference to previous discussions with @wojosc i was settings 'Timesheet' Link in field, if there's something changed regarding this decision then think we should also remove this field 'timesheet'  from 'Timesheet Record'

Reply:
It will be good to have a visible and clickable link from Timesheet Record to Timesheet. As of now I am not confident on which way of doing this would be the best.

Reply:
A link field will work fine for this i guess